1host.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+	port: 3000,
+  modules: [
+    {
+        module: require("./index.js"),
+			"errorHandler": false,
+      config: { usernamee: "wgyt", projectid: "513842800", commentid: "200363559" }
```

Comment:
Codacy has a fix for the issue: [Insert `,`](https://app.codacy.com/gh/onedotprojects/scratchCommentAuth/pullRequest?prid=7364619)
```suggestion
			config: { usernamee: "wgyt", projectid: "513842800", commentid: "200363559" },
```